ansible-ipi-install/README.md

```diff
@@ -418,6 +419,53 @@ With the `playbook.yml` set and in-place, run the `playbook.yml`
 $ ansible-playbook -i inventory/hosts playbook.yml
 ~~~
 
+## Disconnected Registry
+A disconnected registry can be used to deploy the cluster.
+This registry can exist or can be created.
+
+To use a disconnected registry, set the registries host name in 
+the `[registry_host]` group in the inventory file.
+
+### Using an Existing Disconnected Registry
```

Comment:
can we swap the order of existing disconnected and created? I will assume most people wont have one so it be great if we had that first over using existing.

Reply:
yes, that makes sense.

Reply:
done. will be in next commit